rfc_bibtex/rfc_bibtex.py

```diff
@@ -35,8 +36,32 @@ def __init__(self, id_names=[], in_file_name=None, out_file_name=None):
         if in_file_name is not None:
             self._id_names += self._read_ids_from_file(in_file_name)
 
+    @staticmethod
+    def _rfc_key_function(name):
+        """Turn RFC32 into RFC00032 for sorting"""
+        try:
+            if name.startswith('RFC'):
```

Comment:
What if it starts with `rfc` or  even `rFc`? I think this should be case-insensitive